pdns/lua-record.cc

```diff
@@ -738,14 +738,15 @@ static void setupLuaRecords(LuaContext& lua)
       } else if (parts.size() >= 1) {
         // either hex string, or 12-13-14-15
         vector<string> ip_parts;
+        auto input = parts[0];
```

Comment:
I *think* you want `auto& input` otherwise you get a copy ?

Reply:
Also even if we check the size above, `.at()` would be preferable on a vector.

Reply:
> I _think_ you want `auto& input` otherwise you get a copy ?

right, in an earlier version I wanted to leave `parts[0]` intact, but that's no longer true.